src/DataExplorer/index.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import multisort from 'multisort';
+import DataExplorerTable from '../components/tables/DataExplorerTable/.';
+
+// Skeleton for connecting GraphQL data
+const dummyConfig = {
+  timestamp: '2018-01-12T16:42:07.495Z',
+  type: 'subjects',
+  keyField: 'name',
+  defaultSorted: [{ id: 'name', desc: false }],
+  columns: [
+    {
+      show: true,
+      Header: 'Case.ID',
+      type: 'string',
+      sortable: true,
+      canChangeShow: true,
+      accessor: 'name',
+    },
+    {
+      show: true,
+      Header: 'Project',
+      type: 'string',
+      sortable: true,
+      canChangeShow: true,
+      accessor: 'project',
+    },
+    {
+      show: true,
+      Header: 'Study',
+      type: 'string',
+      sortable: true,
+      canChangeShow: true,
+      accessor: 'study',
+    },
+    {
+      show: true,
+      Header: 'Gender',
+      type: 'string',
+      sortable: true,
+      canChangeShow: true,
+      accessor: 'gender',
+    },
+    {
+      show: true,
+      Header: 'Ethnicity',
+      type: 'string',
+      sortable: true,
+      canChangeShow: true,
+      accessor: 'ethnicity',
+    },
+    {
+      show: true,
+      Header: 'Race',
+      type: 'string',
+      sortable: true,
+      canChangeShow: true,
+      accessor: 'race',
+    },
+  ],
+};
+
+const fetchGQL = ({ sort, offset, first }) => fetch(
```

Comment:
write the `fetchGQL` in corresponding `action.js` file.
we have for each page in the portal:
```
- *.less
- *.jsx
- reduxer.js -> mapping redux values to properties
- reducer.js -> if you need some reduce functions for this page
- action.js -> if you need customized fetch, otherwise, you can use some default one in src/actions.js
```